test/prototype_transforms_kernel_infos.py

```diff
@@ -1908,7 +2026,12 @@ def sample_inputs_adjust_gamma_video():
             sample_inputs_fn=sample_inputs_adjust_gamma_image_tensor,
             reference_fn=pil_reference_wrapper(F.adjust_gamma_image_pil),
             reference_inputs_fn=reference_inputs_adjust_gamma_image_tensor,
-            closeness_kwargs=DEFAULT_PIL_REFERENCE_CLOSENESS_KWARGS,
+            closeness_kwargs={
+                # TODO: This tolerance is noticeably larger than "regular" ones. Investigate if this is a bug in the
+                #  kernel for floating point inputs, given that uint8 works almost perfectly.
+                (("TestKernels", "test_against_reference"), torch.float32, "cpu"): dict(atol=40 / 255, rtol=0),
+                (("TestKernels", "test_against_reference"), torch.uint8, "cpu"): dict(atol=1, rtol=0),
```

Comment:
`adjust_gamma_image_tensor`. Since uint8 is fine, maybe this just happens due to our conversion to uint8 and back for PIL. Meaning, this could be a limitation of the test suite. Maybe it is sufficient to only check uint8 here against PIL and separately check that our kernel is consistent for float32 and uint8.